pre_commit_dbt/check_exposure_has_owner.py

```diff
@@ -0,0 +1,40 @@
+from pre_commit_dbt.utils import get_filenames, get_exposures
+from typing import Sequence
+from pathlib import Path
+from typing import Optional
+import argparse
+from pre_commit_dbt.utils import add_filenames_args
+
+    
+"""
+Checks if exposures have owners with a name field defined
+Arguments:
+    paths: a sequence of paths
+Returns:
+    A status code (0 if the defined exposures have owners with a defined name field and 1 if not)
+"""
+def has_owner(paths: Sequence[str]):
```

Comment:
For example here, instead of typing "returns: ...." you can juste add some type hinting with the right type of the variable expected and a good name for it.